ci/build_wheel.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name=$1
+package_dir=$2
+
+# Use gha-tools rapids-pip-wheel-version to generate wheel version then
+# update the necessary files
+RAPIDS_EPOCH_TIMESTAMP=$(date +%s)
+versioneer_override="$(rapids-pip-wheel-version ${RAPIDS_EPOCH_TIMESTAMP})"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CTK_VERSION})"
+
+bash ci/release/apply_wheel_modifications.sh ${versioneer_override} "-${RAPIDS_PY_CUDA_SUFFIX}"
+echo "The package name and/or version was modified in the package source. The git diff is:"
+git diff
+
+cd "${package_dir}"
+
+# sccache configuration
+export SCCACHE_S3_KEY_PREFIX=gha-cibw
+export SCCACHE_REGION=us-east-2
+export SCCACHE_IDLE_TIMEOUT=32768
+export SCCACHE_BUCKET=rapids-sccache-east
+export SCCACHE_S3_USE_SSL=true
```

Comment:
Use the env vars set upstream in the image. The only one you might need is `SCCACHE_S3_KEY_PREFIX`.

We use a different key prefix for each repo in conda -- e.g. `libraft-aarch64` rather than `gha-cibw`. Should we do the same here?

Upstream vars:
https://github.com/rapidsai/cibuildwheel-imgs/blob/fa583bdcb03693a1d8b77f96550b07cf5228dbd2/manylinux_v2_ubuntu/Dockerfile#L29-L37

conda just passes these through, only setting values for the key prefix:
https://github.com/rapidsai/raft/blob/42c9c180f2dcdb1437c45bfb0c753db49ec0a732/conda/recipes/libraft/meta.yaml#L33-L39

Reply:
This is a good point. I can set this up!

Reply:
Okay, I made this change. First run is running through now and it seems like we are not getting any `sccache` hits. Once I run again, we'll know if this worked.

Reply:
Removing these was the goal of https://github.com/rapidsai/cibuildwheel-imgs/pull/21 :slightly_smiling_face: